Review.java

```diff
@@ -204,5 +204,46 @@ else if (totalSentiment(fileName) <= 8)
       {
             return 5;
       }
+    
+    public static String fakeReview(String fileName)
+      {
+        String review = textToString(fileName);
+        String word = "";
+        String sentence = "";
+        for (int i = 0; i < review.length(); i++)
+          {
+            if (review.substring(i, i+1).equals(" ") || i == review.length() - 1)
+            {
+              if (i == review.length() -1)  word += review.substring(i, i+1); // adds the last character in the file to the word. 
+              
+              if (word.startsWith("*"))
+              {
+                String newAdj = "";
+                while (newAdj.equals(""))
+                    newAdj = randomPositiveAdj();

```

Comment:
Instead of just looking for a positive adjective, you can use the method randomAdjective() to have either negative or positive.

Reply:
I wasn't sure if the assignment wanted random Adjectives or improvement adjectives relative to the total sentiment value of the review. But thank you for the comment!